### ISSUE TITLE
Eliminate the Browser - GitHub as a Native App Integrated with Desktop.

### ISSUE THREAD

**Issue by NotMoni** [desktop/desktop#9236]
### Describe the feature or problem you’d like to solve
An easier way to browse Github.com I use the browser to look at repo that i want to contribute to. It would be easier if I could browse and look at repos in Github Desktop. I want to bring the Website to the Desktop App. So i don't have to make an issue (like this)on the browser but can use a Native app. An easier way to look at GitHub Actions, GitHub Projects in a Native Desktop Github App.

### Proposed solution

No Browser. Everything you do on the website can be on a native app with a Git GUI. It's perfect. You can view GitHub projects, GitHub actions, pull requests, issues, insights, all in a native app.

**TLDR: Eliminate the need for a Browser.**

### Additional context

![Screen Shot 2020-03-08 at 9 25 03 AM](https://user-images.githubusercontent.com/40552237/76163684-b2820500-611e-11ea-8954-877b779c601d.png) 
### A start-up page like this in Dark would also help many users who don't prefer Light Mode.
![Screen Shot 2020-03-08 at 9 28 02 AM](https://user-images.githubusercontent.com/40552237/76163732-1c021380-611f-11ea-9d82-ce6986a4a4dd.png)

**Comment by niik** [comment]
Hi @NotMoni, thanks for contributing your ideas! We don't have any plans of replicating the full website experience inside of GitHub Desktop. We believe that the best use of our limited resources is to focus on providing the best possible experience in the areas that a Desktop application excels, i.e. dealing with local content and facilitating the synchronization between local and remote.

We appreciate your suggestion but seeing as this is not something that's in line with the product vision for GitHub Desktop I'm going to close this issue.
